examples/express-ex/index.js

```diff
@@ -1,22 +1,48 @@
-/* eslint-disable no-console */
+/* eslint-disable */
 
 const express = require("express");
-const {Receiver} = require("cloudevents");
-
+const { Receiver, DiscoveryService } = require("cloudevents");
```

Comment:
Can the discovery sample be in a separate example folder?

Folks wishing to set up a server to receive events are likely different than those who want to discover events.

Reply:
Yep i will split